services/front/package.json

```diff
@@ -5,18 +5,18 @@
     "description": "ZKorum frontend",
     "scripts": {
         "dev": "vite",
-        "dev:force": "vite --force",
-        "build": "tsc && vite build",
-        "build:staging1": "tsc && NODE_ENV=development vite build --mode staging1",
+        "build:staging1": "tsc && NODE_ENV=production vite build --mode staging1",
```

Comment:
you can keep this, NODE_ENV=production makes sense, but put this line after "build" command